packages/Angular/Explorer/explorer-core/src/lib/app-view/application-view.component.ts

```diff
@@ -372,7 +372,7 @@ export class ApplicationViewComponent extends BaseBrowserComponent implements On
                 this.sharedService.CreateSimpleNotification('There was an error linking to the shared view(s). Please try again later or notify a system administrator.', 'error', 3500);
             }
         }
-    }
+    } 
```

Comment:
@JS-BC  seems like a non-change unless I'm missing something here?

Reply:
Yes, I just made an empty change here for the purposes of running `npm run change` off of a single file